Rakefile

```diff
@@ -19,4 +19,44 @@ desc 'Generate code coverage'
 RSpec::Core::RakeTask.new(:coverage) do |t|
   t.rcov = true
   t.rcov_opts = ['--exclude', 'spec']
-end
\ No newline at end of file
+end
+
+desc 'Executes reference tests (agent only) intended for use in CI'
+task :reference_tests do
+    command =<<-EOS
+bundle exec beaker                          \
+    --debug                                 \
+    --preserve-hosts never                  \
+    --config tests/configs/$PLATFORM        \
+    --keyfile ~/.ssh/id_rsa-acceptance      \
+    --load-path tests/lib                   \
+    --type aio                              \
+    --pre-suite tests/reference/pre-suite   \
+    --tests tests/reference/tests
+    EOS
+    sh command
+end
+
+desc 'Executes accetpance tests (master and agent) indened for use in CI'
+task :acceptance_tests do
+    command =<<-EOS
+bundle exec beaker
+    --debug                                 \
+    --preserve-hosts never                  \
+    --config tests/configs/$PLATFORM        \
+    --keyfile ~/.ssh/id_rsa-acceptance      \
+    --load-path tests/lib                   \
+    --type aio                              \
+    --pre-suite tests/acceptance/pre-suite  \
+    --tests tests/acceptance/tests
+    EOS
+    sh command
+end
+
+task :acceptance_tests => [:blah]
+task :reference_tests => [:blah]
+
+task :blah do
```

Comment:
This should have a more descriptive name and maybe a comment on how it is triggered.